docs/camino-messenger/introduction.md

```diff
@@ -72,48 +74,70 @@ A blockchain is product or service agnostic. Conclusively our strategy is to kee
+
+1. Camino Messenger Network Fee
+   The Network Fee goes towards the operators of the Messenger Server. Initially the fee is set to 0.01 CAM per message. The fee is the exact same on each server on the network and will be decided by the server operators through voting. The network fee is split between the operator (70%) and Chain4Travel (30% for further development). Instead of working with an anonymous messenger server, we encourage suppliers to host their own messenger server to earn the network fee themselves.
 
-Two messages have been made available to enable automatic detection of changed partners, their settings and fees:
+Each message, that means each request and each response, requires the Network Fee. for example an OTA (distributor) sends a message request (0.01 CAM per message) to 10 Accommodation Suppliers. 7 send a respond message (each pay 0.01 CAM per message).
 
-1. Partner Request gives you all active Camino Network Partners that are selling their products and services. It specifies the messages they support, the Supplier Fee for each message to be paid (if any) and whether the partner support on-chain payment, off-chain payment or both.
-2. The Network Fee Request gives you the cost of a Message to the Camino Network for operating the Camino Messenger Servers. This fee goes towards the Partners operating a Messenger Server.
+2. Camino Messenger Service Fee
+   The Service Fee is set by the supplier for each individual service. It is specified in CAM and can have any value including 0. It's purpose is an additional income stream, balancing out traffic/processing costs, replacement for booking fees or payment for providing information services. The Service Fee is split between the Supplier (70%) and Chain4Travel (30% for further development).
 
-Because this information is stored on the blockchain, the changes since a certain date/time can be requested using a block-height parameter. This is required as not every node on the blockchain is updated at the exact same timestamp. Conclusively, if this information is used in the distribution system, store the block height of the last block you visited. Use the Partner Request to query the C-Chain for blocks that have a higher block height than the last block height you visited. The MessageType filters the transactions to only include those that interact with the ones defined in Partner Configuration. You filter for just the suppliers you are interested in. Process the transaction and update your application state accordingly. Finally update the last block height that you visited to the block height of the last block you processed. The same in case you store the NetworkFee externally.
+The Partner and Network messages will be made available to enable automatic detection of changed partners, their settings and fees.
 
-Figure 2: Supplier Fee and Network Fee
+Figure 2: Camino Messenger Service Fee and Network Fee
```

Comment:
This is confusing. I think it should say "Service Fee and Network Fee" only. 

If we put "Camino Messenger" before the Service Fee than it sounds like Network Fee is not related to Camino Messenger. Remember the last meetings we have, people already started to confuse "Network Fee" with the transaction fees you pay to the network.

I still think "Network Fee" is not a good term here. It should be "Messenger Fee" because it is paid to the messengers (Matrix Messenger operators).

Reply:
We can discuss to change the wording after TU2024 to avoid discrepancies between the different presentations and documentation (code)